src/NuGet.Core/NuGet.Packaging/PackageArchiveReader.cs

```diff
@@ -381,7 +381,16 @@ public override Task<byte[]> GetArchiveHashAsync(HashAlgorithmName hashAlgorithm
         public override bool CanVerifySignedPackages(SignedPackageVerifierSettings verifierSettings)
         {
 #if IS_SIGNING_SUPPORTED
-            return true;
+            // Mono support has been deprioritized, so verification on Mono is not enabled
```

Comment:
Can you link the issue tracking enabling verification on Mono?

Reply:
Thanks for review! Added the link.